src/app.ts

```diff
@@ -185,6 +185,7 @@ export class App {
   validator = new Validator();
 
   _contextClass: any = class extends ContextWrapper {};
+  _nestedFunctions: any = {};
```

Comment:
This can probably be a bit better than `any`.

Reply:
fixed, like Record<string, Record<string, MojoAction>>